service/storage_service.cc

```diff
@@ -497,7 +497,7 @@ future<> storage_service::join_token_ring(cdc::generation_service& cdc_gen_servi
                 // Sleep additionally to make sure that the server actually is not alive
                 // and giving it more time to gossip if alive.
                 slogger.info("Sleeping before replacing {}...", *replace_addr);
-                co_await sleep_abortable(service::load_broadcaster::BROADCAST_INTERVAL, _abort_source);
+                co_await sleep_abortable(2 * get_ring_delay(), _abort_source);
```

Comment:
What would the comment be? There's already a comment above: "Sleep additionally blah blah blah". There was basically no reason to use `service::load_broadcaster::BROADCAST_INTERVAL` before instead of `ring_delay`. And there's more code below with another sleep, this time it's using `ring_delay`. The only reason I'm multiplying it by `2` is legacy (to preserve the behavior for default value of `ring_delay`). Which is not a very good reason TBH. Getting rid of the multiplication would probably work as well. I.e. the `2 *` only makes sense in the context of the change, it doesn't make much sense when you look at the final code.

Reply:
Any such sleeps should go away eventually. I am fine with this change. It is good not to change the default value. The extra sleep is there for the bad consequences of replacing a node that is live.